core/src/main/java/tachyon/client/RemoteBlockInStream.java

```diff
@@ -247,6 +247,9 @@ public static ByteBuffer readRemoteByteBuffer(TachyonFS tachyonFS, ClientBlockIn
     try {
       List<NetAddress> blockLocations = blockInfo.getLocations();
       LOG.info("Block locations:" + blockLocations);
+      int hostResolutionTimeout = conf.getInt(Constants.HOST_RESOLUTION_TIMEOUT, 
+          Constants.DEFAULT_HOST_RESOLUTION_TIMEOUT);
+      String localhost = NetworkUtils.getLocalHostName(hostResolutionTimeout);
```

Comment:
I found this block is repeated in many files. 

How about add a method 
`String NetworkUtils.getLocalHostName() { 
      int hostResolutionTimeout = conf.getInt(Constants.HOST_RESOLUTION_TIMEOUT,                  Constants.DEFAULT_HOST_RESOLUTION_TIMEOUT);
      String localhost = NetworkUtils.getLocalHostName(hostResolutionTimeout);
      return localhost;
} 
and replace these blocks with just  `getLocalHostName()` ?